translations/en.json

```diff
@@ -112,7 +102,7 @@
   "per_month": "{{formattedCount}} / month",
   "supporters": "{{formattedCount}} supporters",
   "devs_funded": "{{formattedCount1}} / {{formattedCount2}} devs funded",
-  "based_on_salary": "Based on a median European €{{formattedCount}} software developer salary from developersalary.com",
+  "based_on_salary": "Based on a median European €{{formattedCount}} software developer salary",
```

Comment:
Also removed this dead url. In any case the number is arbitrary, it would be much higher if we took the average salary for Rust or SQL devs.